lib/Installation/Registration/RegistrationPage.pm

```diff
@@ -25,7 +25,7 @@ sub init {
 
 sub is_shown {
     my ($self) = @_;
-    return $self->{rdb_skip_registration}->exist();
+    return $self->{txb_reg_code}->exist();
```

Comment:
we will need a verification for installation to be sure we don't break anything.

Reply:
Already verify on current case, no regression happen.